101-traffic-manager-webapp/azuredeploy.json

```diff
@@ -0,0 +1,99 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "dnsname": {
+            "type": "string",
+            "metadata": {
+                "description": "Relative DNS name for the traffic manager profile, resulting FQDN will be <dnsname>.trafficmanager.net, must be globally unique."
+            }
+        },
+        "dnsnameForWebApp": {
+            "type": "string",
+            "metadata": {
+                "description": "Relative DNS name for the WebApps, must be globally unique.  An index will be appended for each Web App."
+            }
+        }
+    },
+
+    "variables": {
+        "tmApiVersion": "2015-04-28-preview",
+        "apiVersion": "2015-08-01",
+        "locations": [ "westus", "eastus" ],
```

Comment:
Updated the region names, also renamed the element to enforce that these are the chosen locations and not a restriction.